.vscode/settings.json

```diff
@@ -17,7 +17,7 @@
   // "explorer.confirmDelete": false,
   // "explorer.confirmDragAndDrop": false,
 
-  "eslint.runtime": "node",
+  // "eslint.runtime": "node",
```

Comment:
Why was it necessary to disable this?

Reply:
This has been causing some issues with package managers (like `nvs` that I use). Basically, ESLINT doesn't find the correct node executable if I have it set like that. I thing it is a bug and it is reported here:
https://github.com/microsoft/vscode-eslint/issues/1596

Reply:
Makes sense! 

Personally, I would delete most of it and prefer to keep VSCode settings to as minimal as possible. It's already enforcing some options that I don't like and all of it could be configured by user itself based on preference without affecting every dev.